utils/npm-publish/npm-publish.js

```diff
@@ -56,7 +57,7 @@ function npmPublish(pkg, tarFilePath, _opts) {
         manifest.publishConfig.tag = opts.tag;
       }
 
-      return publish(manifest, tarData, opts).catch(err => {
+      return otplease(opts => publish(manifest, tarData, opts), opts, otpCache).catch(err => {
         opts.log.silly("", err);
```

Comment:
Perhaps the logging problems are related to the default `npmlog` export and `this.logger` from the command instance? I've previously had problems with one or the other not _quite_ responding consistently to "shut up now" and "ok spew now" methods...